README.md

```diff
@@ -1,2 +1,35 @@
-# to-do-list
+# To Do List
 this repo created to contains to do list project files.
+
+## Built With
+- html
+- css
+- js
+## Live Demo (if available)
+Live Demo [Link](#)
+## Getting Started
+Simply you can open index.html file with any browser and, you will see the home page.
+for opennig index.html go to to-do-list directory, find index.html, right click on it, click on open with and, click on any browser which is available on your pc/labtop. 
```

Comment:
- [OPTIONAL] I think it's better to explain how someone who wants to check your application all the steps to execute your code. How to clone your repository, run `npm install` and `npm start`. The recruiters and hiring managers can check your work easily.